free/views/api.py

```diff
@@ -70,7 +70,7 @@ class ExecutionUpdateSerializer(serializers.ModelSerializer):
     def validate(self, data):
         data = super().validate(data)
             
-        if not self.instance.status in ['C','N']:
+        if not self.instance.status in ['C','N','F']:
```

Comment:
@PedroRossa98 should we add the new endpoint in this PR or in another iteration?